tests/l2/README.md

```diff
@@ -61,6 +64,76 @@ $ oc logs intel-dgpu-clinfo-56mh2
   Device OpenCL C all versions                    OpenCL 
 ```                                               
 
+### Verify Intel® QuickAssist Technology provisioning
+This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details. 
+
+*	Build the workload container image. 
+
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/qat/qatlib_build.yaml ```
+
+*	Deploy and execute the workload.
+
+    * Run with IPC_LOCK capability for pod
+  
```

Comment:
Can we change it to 
Create the intel-qat service account to run the qatlib based workload